src/ng-select/lib/ng-dropdown-panel.component.ts

```diff
@@ -118,19 +96,17 @@ export class NgDropdownPanelComponent implements OnInit, OnChanges, OnDestroy {
 	}
 
 	private get _startOffset() {
-		if (this.markedItem) {
+		const markedItem = this.markedItem();
```

Comment:
@pankajparkar I see new variables like this in many places. We can just use existing format. Any reason for change?

Reply:
Updated - https://github.com/ng-select/ng-select/pull/2584/commits/53cf59e5182d0cbc93ac9608a82ee5b0dc0c44be